src/main/java/com/github/seratch/jslack/api/methods/impl/MethodsClientImpl.java

```diff
@@ -1,27 +1,96 @@
 package com.github.seratch.jslack.api.methods.impl;
 
+import static java.util.stream.Collectors.joining;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 import com.github.seratch.jslack.api.methods.Methods;
 import com.github.seratch.jslack.api.methods.MethodsClient;
 import com.github.seratch.jslack.api.methods.SlackApiException;
 import com.github.seratch.jslack.api.methods.request.api.ApiTestRequest;
 import com.github.seratch.jslack.api.methods.request.auth.AuthRevokeRequest;
 import com.github.seratch.jslack.api.methods.request.auth.AuthTestRequest;
 import com.github.seratch.jslack.api.methods.request.bots.BotsInfoRequest;
-import com.github.seratch.jslack.api.methods.request.channels.*;
+import com.github.seratch.jslack.api.methods.request.channels.ChannelsArchiveRequest;
```

Comment:
hmm, can you adjust your IDE settings not to expand wildcard imports? I'm using IntelliJ IDEA with default settings.